en/simulation/gazebo.md

```diff
@@ -258,19 +260,58 @@ The next time you build/restart Gazebo it will use the new GPS noise setting.
 
 By default Gazebo displays a flat featureless plane, as defined in [empty.world](https://github.com/PX4/sitl_gazebo/blob/master/worlds/empty.world).
 
-You can load any of the worlds in [PX4/sitl_gazebo/worlds](https://github.com/PX4/sitl_gazebo/tree/master/worlds)) by specifying them as the final option in the PX4 configuration target.
+You can load any of the worlds in [PX4/sitl_gazebo/worlds](https://github.com/PX4/sitl_gazebo/tree/master/worlds) by specifying them as the final option in the PX4 configuration target:
+```
+make px4_sitl gazebo_<model_name>__<world_name>
+```
+
 For example, to load the *warehouse* world, you can append it as shown:
 ```
 make px4_sitl_default gazebo_plane_cam__warehouse
 ```
 
-> **Note** There are two underscores after the model (`plane_cam`) indicating that the default debugger is used (none).
+> **Note** There are *two underscores* after the model (`plane_cam`) indicating that the default debugger is used (none).
    See [Building the Code > PX4 Make Build Targets](../setup/building_px4.md#make_targets).
 
-
 You can also specify the full path to a world to load using the `PX4_SITL_WORLD` environment variable.
 This is useful if testing a new world that is not yet included with PX4.
 
+> **Tip** If the loaded world does not align with the map, you may need to [set the world location](#set_world_location). 
+  
+
+## Set World Location {#set_world_location}
+
+By default, a world is loaded at the current vehicle location.
+If using a world that simulates a real location, this can result in a mismatch between what is displayed from the vehicle camera and the what is shown on the ground station map.
+To overcome this problem you can set the location of the world as shown in this section.
+
+> **Note** You can also set a [Custom Takeoff Location](#custom_takeoff_location) that matches the real-world location of the map.
+  However adding the location to the map is easier (and can still be over-ridden if needed).
+
+The location of the world is defined in the **.world** file by specifying the location of the origin using the `spherical_coordinates` tag.
```

Comment:
What is the origin? i.e. it is clear in the general sense how to align the model, but not in the very specifics. I also assume that the orientation of the model will be right?

Reply:
The origin is (0, 0, 0) of the world - and we can define the gps coordinates(location) of the origin. The orientation of the model doesn't matter, since coordinate in gazebo is ENU(East North Up)

Reply:
OK. See my other clarification. I'm pretty sure I got it - just want to be sure: https://github.com/PX4/Devguide/pull/993/files#r399863369